### PR TITLE
feature/web

### PR DESCRIPTION
**Pull request by pverscha**
This PR adds both a web client written in Vue and a very simple HTTP API (made with Flask) to the project. A Dockerfile that builds a Docker image with everything required to run the web application is also available (and is also present on Docker Hub).

### CODE REVIEW

File: web/src/components/GoListComponent.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            :search="search">
+            <template v-slot:item.name="{ item }">
+                <div style="max-width: 220px; white-space: nowrap; overflow: hidden; text-overflow: ellipsis;" :title="item.name">
+                    {{ item.name }}
+                </div>
+            </template>
+        </v-data-table>
+    </div>
+
+<!--    <v-virtual-scroll v-else height="300" :items="items" :item-height="50" :bench="50">-->
```
Comment (bmesuere) [inline web/src/components/GoListComponent.vue:25]: Is this still here on purpose?

File: web/src/views/About.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (bmesuere) [inline web/src/views/About.vue:1]: still empty?

File: web/src/components/GoListComponent.vue (hunk 1 above)
Comment (pverscha) [reply web/src/components/GoListComponent.vue:25]: It isn't. I'll remove it :)

File: web/src/views/About.vue (hunk 2 above)
Comment (pverscha) [reply web/src/views/About.vue:1]: I'll take a look at providing a basic "about us" page.

**Pull request merged**